test/core/api/extrinsic/extrinsic_api_integration_test.cpp

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "api/extrinsic/service.hpp"
+
+#include <chrono>
+#include <thread>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
```

Comment:
no need for blank line here

Reply:
removed blank line and some redundant includes